priv/templates/bin

```diff
@@ -11,30 +11,30 @@ REL_DIR="$RELEASE_ROOT_DIR/releases/$REL_VSN"
 ERL_OPTS="{{ erl_opts }}"
 
 find_erts_dir() {
-    local erts_dir="$RELEASE_ROOT_DIR/erts-$ERTS_VSN"
-    if [ -d "$erts_dir" ]; then
-        ERTS_DIR="$erts_dir";
+    l_erts_dir="$RELEASE_ROOT_DIR/erts-$ERTS_VSN"
```

Comment:
I mostly did that because in the block above `erl` was the var name and I didn't want that to get confused with `erl` the binary (that is in the path).  So I just kept the pattern throughout.